docs/basic-features/eslint.md

```diff
@@ -261,3 +261,5 @@ If you already use a separate ESLint configuration and want to include `eslint-c
 ```
 
 The `next` configuration already handles setting default values for the `parser`, `plugins` and `settings` properties. There is no need to manually re-declare any of these properties unless you need a different configuration for your use case. If you include any other shareable configurations, **you will need to make sure that these properties are not overwritten or modified**. Otherwise, we recommend removing any configurations that share behavior with the `next` configuration or extending directly from the Next.js ESLint plugin as mentioned above.
+
+If ESLint shows a conflicting plugin error, it is because you have installed a different version than what Next.js already includes. You should remove the conflicting plugins from your `package.json`.
```

Comment:
I think our usage of [@rushstack/eslint-patch](https://www.npmjs.com/package/@rushstack/eslint-patch) prevents us from using a typical `peerDependencies` setup in favor of simplifying the workflow for developers (they only need to install a single package (`eslint-config-next`) instead of all the plugins separately as well).

I'd be happy to consider changing this if there are issues with this approach, but I agree it would have to be in a major version change.

Reply:
Yeah. Our current approach seems to cause issues: https://github.com/vercel/next.js/issues/35806

https://www.npmjs.com/package/eslint-config-airbnb recommends `npx install-peerdeps --dev eslint-config-airbnb` which will auto-install all the necessary `peerDependencies`. Could that be an option that we could bake in?

Reply:
> Yeah. Our current approach seems to cause issues (see https://github.com/vercel/next.js/issues/35806 also) and I can see it being the ground for more the future.

In that case, then let's aim to switch it to a normal `peerDep` setup. :)

> Could that be an option that we could bake in?

Yep that looks like the best possible path forward. If possible, I would like to bake this option in first and then show a warning for folks who don't have `peerDeps` installed. That will give them enough time to switch before we introduce the breaking change of removing [@rushstack/eslint-patch](https://www.npmjs.com/package/@rushstack/eslint-patch) completely.

Reply:
So to resolve https://github.com/vercel/next.js/issues/35606 and https://github.com/vercel/next.js/issues/35977 for now do we want to unpin the plugin deps in this PR and work towards moving these to `peerDeps`?